syntax = "proto2";
package libfuzzer_example;

message Msg {
  optional bool optional_bool = 1;
  optional string optional_string = 2;
}
